packages/cli/README.md

```diff
@@ -29,6 +29,18 @@ Loads input files a local filesystem path.
 * `sourceDir`
   * default: `/input`
 
+### `--from s3`
+
+Loads input files from an S3 bucket.
+
+#### Arguments
+
+* `S3-ENDPOINT`
```

Comment:
Any reason to have these in all-caps?

Reply:
Hm, no reason. Copied from env variables the API uses. Lower case would be more consistent?

Reply:
Since all the other arguments are lowercase, I would say yes